tests/c/test_livefiles.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from zeroae.rocksdb.c import db as rdb
+import zeroae.rocksdb.c.livefiles as rlf
+
```

Comment:
Because we import all of "db" into rocksdb.c we end up clobbering the `livefiles` module with `rdb.livefiles` function.